lib/tasks/user_stats.rake

```diff
@@ -0,0 +1,10 @@
+task user_stats: 'kitsu:user_stats_setup'
```

Comment:
uhhhh why are you copying `stats.rake`? That's literally just a thing to configure how we count lines of code with `rake stats` (try running it, it'll tell you the lines of code in Kitsu!)

It shouldn't be named `user_stats_setup` unless it's doing some from of configuration setup (which it isn't)